test/EndToEnd/ValidateInsertedManifests.cs

```diff
@@ -31,10 +31,14 @@ public void ManifestReaderCanReadManifests()
 
                     string manifestFile = manifestDir.GetFile("WorkloadManifest.json").FullName;
 
-                    File.Exists(manifestFile).Should().BeTrue();
-                    using var fileStream = new FileStream(manifestFile, FileMode.Open, FileAccess.Read);
-                    Action readManifest = () => WorkloadManifestReader.ReadWorkloadManifest(manifestId, fileStream, manifestFile);
-                    readManifest.ShouldNotThrow("manifestId:" + manifestId + " manifestFile:" + manifestFile + "is invalid");
+                    var baselineFile = manifestDir.GetFile("Baseline.WorkloadSet.json").FullName;
```

Comment:
The casing here should probably be lowercase too.

```suggestion
                    var baselineFile = manifestDir.GetFile("baseline.workloadset.json").FullName;
```